server.js

```diff
@@ -1,37 +1,35 @@
-const express = require('express');
-const bodyParser = require('body-parser')
-const path = require('path');
+const express = require("express");
+const bodyParser = require("body-parser");
+const path = require("path");
 const app = express();
-const tokenGenerator = require('./token_generator');
+const tokenGenerator = require("./token_generator");
 
 // load configuration from .env if available
-require('dotenv').config();
+require("dotenv").config();
```

Comment:
Do we still need this if you took out the `process.env.PORT`?

Reply:
Yes, it pulls in the API Keys, service SIDs, etc. from the .env file.

Reply:
The config.js file also calls it, though?

Reply:
Oh, I see... we're calling it twice. Good catch. I'll push up a fix.